src/aliceVision/mesh/Texturing.cpp

```diff
@@ -451,7 +451,7 @@ void Texturing::generateTexturesSubSet(const mvsUtils::MultiViewParams& mp,
                 if(contrib + 1 == nbContribLevel)
                 {
```

Comment:
Few lines above, we have:
`int nbContribMax = std::min(texParams.multiBandNbContrib.back(), static_cast<int>(scorePerCamId.size()));`

I think yes, but just to be sure: Does `texParams.multiBandNbContrib` still contains cumulative values in the latest version?